graylog2-web-interface/test/setup-jest.jsx

```diff
@@ -9,3 +9,7 @@ global.jQuery = jQuery;
 registerBuiltinStores();
 
 configure({ adapter: new Adapter() });
+
+console.error = jest.fn((error) => {
+  throw new Error(error);
```

Comment:
Unfortunately this does not include all arguments passed to `console.error` and omits string interpolation. Using `util.format` should already fix this here.